packages/acs-ui-common/src/cssUtils.ts

```diff
@@ -7,3 +7,16 @@
  * For example, an input of `16` will return `1rem`.
  */
 export const _pxToRem = (px: number): string => `${px / 16}rem`;
+
+/**
+ * @internal
+ * Converts rem value to px value.
+ * For example, an input of `1rem` will return `16`.
+ */
+export function _remToPx(rem: string | number, baseFontSize: number = 16): number {
```

Comment:
We don't want to have this function as we want All of our CSS to be given to the DOM as `rem` for responsivness and scaling when the end user affects the page with zoom.